### PR TITLE
Add four stub packages

### PR DESCRIPTION
**Pull request by GeoffCoxMSFT**
## Changes
- Added stub merge-styles
- Added stub  react-cards
- Added stub react-hooks
- Added stub scheme-utilities

## Issues
Updates #24393

### CODE REVIEW

**Comment by fabricteam** [comment]
## [Perf Analysis](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">7.0 Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/BaseButton-mount_base.html">679</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/BaseButton-mount.html">708</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/Breadcrumb-mount_base.html">32321</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/Breadcrumb-mount.html">32482</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/Checkbox-mount_base.html">1305</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/Checkbox-mount.html">1215</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/CheckboxBase-mount_base.html">958</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/CheckboxBase-mount.html">962</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/ChoiceGroup-mount_base.html">3797</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/ChoiceGroup-mount.html">3723</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/ComboBox-mount_base.html">650</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/ComboBox-mount.html">681</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/CommandBar-mount_base.html">5973</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/CommandBar-mount.html">5981</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/ContextualMenu-mount_base.html">12073</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/ContextualMenu-mount.html">12082</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/DefaultButton-mount_base.html">877</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/DefaultButton-mount.html">860</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/DetailsRow-mount_base.html">2867</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/DetailsRow-mount.html">2985</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/DetailsRowFast-mount_base.html">2806</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/DetailsRowFast-mount.html">2869</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/DetailsRowNoStyles-mount_base.html">2724</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/DetailsRowNoStyles-mount.html">2746</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/Dialog-mount_base.html">1795</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/Dialog-mount.html">1729</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/DocumentCardTitle-mount_base.html">1582</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/DocumentCardTitle-mount.html">1576</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/Dropdown-mount_base.html">1891</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/Dropdown-mount.html">1999</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/FocusTrapZone-mount_base.html">1344</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/FocusTrapZone-mount.html">1435</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/FocusZone-mount_base.html">1406</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/FocusZone-mount.html">1502</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/IconButton-mount_base.html">1412</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/IconButton-mount.html">1372</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/Label-mount_base.html">284</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/Label-mount.html">254</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/Layer-mount_base.html">1471</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/Layer-mount.html">1571</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/Link-mount_base.html">329</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/Link-mount.html">328</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/MenuButton-mount_base.html">1106</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/MenuButton-mount.html">1101</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/MessageBar-mount_base.html">1717</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/MessageBar-mount.html">1602</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/Nav-mount_base.html">2498</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/Nav-mount.html">2558</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/OverflowSet-mount_base.html">1083</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/OverflowSet-mount.html">1070</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/Panel-mount_base.html">1172</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/Panel-mount.html">1139</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/Persona-mount_base.html">787</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/Persona-mount.html">808</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/Pivot-mount_base.html">1147</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/Pivot-mount.html">1261</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/PrimaryButton-mount_base.html">1003</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/PrimaryButton-mount.html">1001</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/Rating-mount_base.html">6115</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/Rating-mount.html">6190</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/SearchBox-mount_base.html">1034</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/SearchBox-mount.html">931</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/Shimmer-mount_base.html">2029</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/Shimmer-mount.html">1966</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/Slider-mount_base.html">1121</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/Slider-mount.html">1126</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/SpinButton-mount_base.html">3887</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/SpinButton-mount.html">3987</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/Spinner-mount_base.html">302</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/Spinner-mount.html">311</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/SplitButton-mount_base.html">2384</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/SplitButton-mount.html">2411</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/Stack-mount_base.html">421</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/Stack-mount.html">369</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/StackWithIntrinsicChildren-mount_base.html">1396</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/StackWithIntrinsicChildren-mount.html">1292</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/StackWithTextChildren-mount_base.html">3800</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/StackWithTextChildren-mount.html">3785</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/SwatchColorPicker-mount_base.html">8138</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/SwatchColorPicker-mount.html">8347</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/TagPicker-mount_base.html">2269</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/TagPicker-mount.html">2209</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/TeachingBubble-mount_base.html">39221</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/TeachingBubble-mount.html">39160</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/Text-mount_base.html">318</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/Text-mount.html">308</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/TextField-mount_base.html">1024</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/TextField-mount.html">1040</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/Toggle-mount_base.html">618</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/Toggle-mount.html">630</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/button-mount_base.html">84</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24628/perf-test/button-mount.html">89</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

File: stub-packages/merge-styles/src/DeepPartial.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export * from '@uifabric/merge-styles/lib/DeepPartial';
```
Comment (micahgodbolt) [inline stub-packages/merge-styles/src/DeepPartial.ts:1]: DeepPartial and other exports not listed in v8 merge-styles package.json/exports should not be included here.
https://github.com/microsoft/fluentui/blob/master/packages/merge-styles/package.json#L34-L70
image

File: stub-packages/react-hooks/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,2 @@
+import './version';
+export * from '@uifabric/react-hooks';
```
Comment (micahgodbolt) [inline stub-packages/react-hooks/src/index.ts:2]: This index file should be the only export for react-hooks to match exports of the v8 version.

**Review by micahgodbolt — CHANGES_REQUESTED** [review]
remove src files that are not included in the v8 version's package.json/exports

File: stub-packages/react-hooks/src/testUtilities.tsx [hunk 3]
```diff
@@ -0,0 +1 @@
+export * from '@uifabric/react-hooks/lib/testUtilities';
```
Comment (micahgodbolt) [inline stub-packages/react-hooks/src/testUtilities.tsx:1]: is this in here for a reason?

File: stub-packages/merge-styles/src/index.ts [hunk 4]
```diff
@@ -0,0 +1,2 @@
+import './version';
```
Comment (micahgodbolt) [inline stub-packages/merge-styles/src/index.ts:1]: merge-styles in v8 currently exports the following files
fontFace
IRawStyleBase
IStyle
keyframes
mergeStyles
mergeStyleSets
server"

**Review by micahgodbolt — APPROVED** [review]

**Pull request merged**